desktop/packages/windows/wix/licence.rtf

```diff
@@ -0,0 +1,9703 @@
+{\rtf1\ansi\deff0{\fonttbl{\f0 \fswiss Helvetica;}{\f1 \fmodern Courier;}}
```

Comment:
I think you forgot to remove that file? :sweat_smile:

Reply:
😭